.github/label-actions.yml

```
@@ -475,3 +475,27 @@
 
 
     Thanks, the Renovate team
+
+'auto:enterprise-support':
+  comment: >
+    Hi there,
+
+
+    This discussion has been labeled as a use case specific to "enterprise" users. Enterprise users typically self-host Renovate and have advanced requirements like custom certificates, HTTP proxies, private registries, air gapping, etc.
+
+
+    Such cases usually require substantial back-and-forth to resolve, because the problem relates to the user's specific environment and cannot be reproduced publicly for maintainers to debug.
+
+
+    The maintainers of this project have decided that their time is best spent on issues that benefit large numbers of users. This means that enterprise-specific issues affecting one or a small number of users are usually not a priority. If we can help you quickly, we always will, but if you are seeing this message then a maintainer believes this is not quick and they won't be actively working on this issue unless new information comes to light.
+
+
+    If you are an enterprise user, please consider the following options available to you:
+
+      - You may need to troubleshoot and contribute a fix yourself, or
+      - You may need to [hire a consultant](https://github.com/renovatebot/renovate/blob/main/CODE_OF_CONDUCT.md#getting-more-help) to help you if you cannot, or
+      - You should consider an [Enterprise support contract from Mend.io](https://mend.io/mend-renovate/), or
+      - You may need to wait for a community member to help you if they have the same problem
+
+
+  If you choose to wait, then this discussion can remain open indefinitely, but please be patient and do not nudge or bump this discussion just to get attention.
```

```suggestion
  If you choose to wait, then this discussion can remain open indefinitely.
  But please be patient, do not nudge or bump this discussion just to get attention.
```

Do you _really_ mean for users to leave open discussions lying around?

If we want to leave them open, we should tag them: `looking-for-help` or something like that. Then we can review them over time, and maybe close them if they become really stale?

---

I think we normally leave unanswered discussions open